lib/app/features/chat/recent_chats/providers/conversations_provider.dart

```diff
@@ -46,4 +49,28 @@ class Conversations extends _$Conversations {
       return List<RecentChatDataModel>.unmodifiable(newData);
     });
   }
+
+  void addGroupConversation(Group group) {
+    update((currentData) {
+      final newConversation = RecentChatDataModel(
+        MessageAuthor(
+          name: group.name,
+          imageUrl: 'https://x.com/ice_blockchain/photo',
```

Comment:
Do we have group imageUrl for `Group`?

Reply:
As for now, there is `MediaFile`. I would suggest leave it as it is, cos it's the same as for channels, so it will be changed in one pack, I suppose.

Reply:
A bit refactored this thing, so Group now have image url and can be handled by UI.